zebrad/tests/common/getblocktemplate.rs

```diff
@@ -0,0 +1,164 @@
+use super::*;
```

Comment:
Should we rename this file to `get_block_template_rpcs.rs`?

Reply:
I was considering that. Updated in [`bcc6c28`](https://github.com/ZcashFoundation/zebra/pull/5526/commits/bcc6c28118559f0a48f07837ae88023c37361902)